_posts/blog/2016-12-22-dwh-part1_getting_started.md

```diff
@@ -0,0 +1,134 @@
+
+## Data Lake – why would we need one?
+
+We have already gained more than 6 million monthly unique users and it seems that this number will keep growing. Thus, our data management process is clearly within the realm
+of Big Data. However, relational database systems are usually not well-suited to handle semi-structured and unstructured data as we have to deal with. Consequently, we also
+discussed a more 'Big-Data-friendly' solution which is known as a data lake [^3]. Stein and Morriso describe a data lake as follows: "_A repository for large quantities and
+varieties of data, both structured and unstructured._" [^4]. Based on this definition, a data lake solution could meet our needs perfectly. We have looked at this topic more
+closely, conducted further research, and defined the following components of a data lake: _sources_, _data repository_, and _interactive query service_. The points and diagram
+below describe these components.
+
+1. Sources: As described earlier, we have to deal with structured, semi-structured, and unstructured data.
+2. Data Repository: This is used to store the data heterogeneously. It preserves the original data fidelity extracted from the sources and the lineage of data transformation.
+3. Interactive Query Service: This is a service which accesses the data repository and transforms the data into a format that can be consumed by data science, data mining, and
+   business intelligence experts. The interactive query service can be a transformation rule which defines how to transform the data. This transformation rule might be used to
+   store the results persistently into the data repository or into another data base. It can be also used for exploring the raw data.
+
+<center>
+<picture>
+    <a href="/images/blog/dwh_journey/query_service.png"><img src="/images/blog/dwh_journey/query_service.png" alt="Components of a data lake" width="850"></a>
+    <figcaption>Figure 2: Components of a data lake</figcaption>
+</picture>
+</center>
+
+
+## Adaption of processing steps
+
+The data processing steps are similar to those described in the process chain above and are still valid regarding the data lake approach. In our use case, we adapt the
+process chain in two main ways: storage and interactive query service.
+
+* Storage: For this use case, S3 and HDFS are appropriate storage solutions.
+* Interactive Query Service: Query tools can be Hadoop, Spark, or Amazon Athena.
+
+## Summary and outlook
+
+In this article we briefly introduced upday and described the data we deal with. Further on, we came up with a tentative definition of a DWH and gave a general introduction to
+basic components and processing steps regarding our use case. Finally, we discussed a data lake as a more 'Big-Data-friendly' solution.
+
+This article gives a general overview which is deepened in the next articles. We will describe the first iteration of our DWH implementation and express the selection of tools.
```

Comment:
How about:
"Stay tuned for the next blog posts, where we will describe ... "  instead of the "This article gives a general overview which is deepened in the next articles. We will describe "